en/docs/setup/iam-feature-maintenance-policy.md

```diff
@@ -0,0 +1,99 @@
+# IAM Feature Maintenance Policy
+
+## Feature Deprecation
+
+Features/APIs in the server can be deprecated for several reasons such as
+
+*   New specification versions have been released
+*   Standard alternatives available OOTB
+*   Security risks involved
+
+### Feature Deprecation Life-Cycle
+
+While deprecated features can still be available and support for existing users, they will not be further enhanced, nor recommended for use. Removal of deprecated capabilities will occur over a deprecation window, in which the feature will undergo several phases in the deprecation life-cycle. 
+
+![feature-deprecation-life-cycle](../assets/img/setup/feature-deprecation-life-cycle.png)
+
+The following terms refer to the different phases of the feature deprecation life cycle.
+
+1. **Planned**
+
+    Any feature in the Planned phase, will be deprecated and removed in future releases. However, the feature is available for use in the current release.
+
+2. **Deprecated**
+
+    In the deprecated phase, we no longer recommend using the feature. While deprecated classes and methods are still available, they are tagged deprecated and are disabled by default in the product, because they will be removed in future releases. This is done to facilitate the phasing-out for any existing user that still uses the deprecated features for their use-cases. It is not recommended to use them for implementing any new use cases/extensions. We encourage you to start upgrading any existing code written using the deprecated features during this phase.
+
+3. **Retired**
+
+    At this phase, we decide not to incorporate the feature with new releases. For versions released, until the respective version goes end of license, feature maintenance and security updates will be provided for subscriptions
+
+### Feature Deprecation Matrix
```

Comment:
Shall we bring "Feature deprecation matrix" to the top of the page and make it the main focus? Then we can title the page as "WSO2 Identity Server Feature Deprecation" and it will be clear at a glance why it is under the Migration section of the docs.

Reply:
+1 to change the title to "WSO2 Identity Server Feature Deprecation". 

However I think we need to explain to the users about the deprecation life-cycle before displaying the deprecation matrix (in which the life-cycle timelines of individual features are shown). This can be similar to [1]

WDYT @sherenewso2 @malithie @GANGANI 

[1] https://wso2.com/products/support-matrix/

Reply:
yes let's explain the cycle and then the matrix

Reply:
+1

Reply:
Fixed https://github.com/wso2/docs-is/pull/2205/commits/29aa1c7da1ceb0af5c10e4d0be871e71a3f03f4d